### PR TITLE
fix: avoid to throw error when link doesn't have parentNode

### PR DESCRIPTION
**Pull request by VieZhong**
This PR contains a:

- [x] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

image
The variable ```i``` is a Link tag. But when it loads error, it's parentNode is null. So it will throw an error.

Here is the fix , just like what Alexander does in hmr.
image

### Additional Info

see this issuse: https://github.com/webpack-contrib/mini-css-extract-plugin/issues/1017

### CODE REVIEW

**Comment by VieZhong** [comment]
您好，您的邮件我已经收到。      By 钟远维   联系方式：13098843729    QQ：995825700   新浪微博： 百年佛一__屎维保持谦虚

File: src/index.js [hunk 1]
```diff
@@ -862,7 +862,7 @@ class MiniCssExtractPlugin {
                     'err.code = "CSS_CHUNK_LOAD_FAILED";',
                     "err.type = errorType;",
                     "err.request = realHref;",
-                    "linkTag.parentNode.removeChild(linkTag)",
+                    "if(linkTag.parentNode) linkTag.parentNode.removeChild(linkTag)",
```
Comment (alexander-akait) [inline src/index.js:865]: Please add space ` ` after `if`

Comment (VieZhong) [reply src/index.js]: @alexander-akait I fixed it. Please review again~

**Review by snitin315 — DISMISSED** [review]
LGTM

**Comment by VieZhong** [comment]
> 

I have pushed a commit to fix test case errors. Please approve again! Thx.

**Comment by snitin315** [comment]
[OLD API tests](https://github.com/webpack-contrib/mini-css-extract-plugin/actions/runs/4219652620/workflow#L135) are failing, can you check?

**Comment by VieZhong** [comment]
> [OLD API tests](https://github.com/webpack-contrib/mini-css-extract-plugin/actions/runs/4219652620/workflow#L135) are failing, can you check?

I missed testing the old API. And Now I fixed it. Please review the code again.

**Review by snitin315 — APPROVED** [review]

**Comment by VieZhong** [comment]
@alexander-akait @snitin315

**Comment by alexander-akait** [comment]
Thank you

**Pull request merged**